src/clr-addons/export-datagrid-button/export-datagrid-button.component.ts

```diff
@@ -0,0 +1,118 @@
+import { Component, Output, EventEmitter, input, ElementRef, computed } from '@angular/core';
+import { ClrDatagrid } from '@clr/angular';
+import zipcelx, { ZipCelXConfig } from 'zipcelx';
+import { ClarityModule } from '@clr/angular';
+import { NgIf, NgForOf, NgClass } from '@angular/common';
+import { ExportType, ExportTypeEnum } from './export-type.model';
+
+@Component({
+  selector: 'clr-export-datagrid-button',
+  templateUrl: './export-datagrid-button.component.html',
+  styleUrl: './export-datagrid-button.component.scss',
+  standalone: true,
+  imports: [ClarityModule, NgIf, NgForOf, NgClass],
+})
+export class ExportDatagridButtonComponent {
```

Comment:
Guess we also need to document it on the website?

Reply:
alright from my point of view you can go ahead :)

Reply:
Great, thank you :)